packages/flutter_tools/test/general.shard/build_system/targets/windows_test.dart

```diff
@@ -90,7 +90,7 @@ void main() {
       await buildSystem.build('unpack_windows', environment, const BuildSystemConfig());
 
       expect(fs.file(r'C:\windows\flutter\flutter_export.h').statSync().modified, isNot(modified));
```

Comment:
File modification times don't have high enough resolution on Windows for this sort of test. You can add a delay before the build() call, or use File.setLastModifiedSync to an artificially early time.

Reply:
Today I learned